broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -864,17 +864,6 @@ private RemotingCommand getConsumeStats(ChannelHandlerContext ctx,
                 continue;
             }
 
-            {
-                SubscriptionData findSubscriptionData =
-                    this.brokerController.getConsumerManager().findSubscriptionData(requestHeader.getConsumerGroup(), topic);
-
-                if (null == findSubscriptionData
```

Comment:
findSubscriptionData = null not means the subscription count is 0 because one consumer group can subscribe more than one topic.

Reply:
got it

Reply:
@duhenglucky no as ur expected, as code above, when consumerGroupInfo==null, then findSubscriptionDataCount will return 0.

Reply:
@duhenglucky I know ur idea, but this code should check the case: group is not used, then the findSubscriptionData return null and findSubscriptionDataCount return 0. As the code run, the not used group will return back to user. so maybe we should distinguish two case, one as u say above, and the second as I refer. Do u agree with that?

Reply:
yep, if the topic was set in the request, these codes look a bit messy,  and we need to deal with this in an elegant way

Reply:
@duhenglucky According to discuss above, I update code to distinguish group deleted and sub many topic.